backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -435,7 +435,7 @@ Common::Keymap *SdlGraphicsManager::getKeymap() {
 	};
 
 	for (uint i = 0; i < ARRAYSIZE(filters); i++) {
-		act = new Action(filters[i].id, filters[i].description);
+		act = new Action(filters[i].id, _(filters[i].description));
```

Comment:
Why this change? If you really need to translate description, this is a completely incorrect way of doing it and will break the translation subsystem.

Reply:
Actually your change is correct here. The strings are marked for translation using the `_s()` macro above in the `filters` array definition. However in current master they are not translated (you can see that in the Options dialog) and this is a bug. With your change they are now properly translated in the GUI.

Reply:
Glad that you've cleared it up. I saw by examples before that strings can use `_s` or `_sc`, and then when I actually want to use the translation, I can use `_(...)` like normal, so I was confused as to why this was wrong. Thanks, Resolving this convo.